Colorama is only needed for windows, so it doesn't apply for an RPM.

Sent upstream as https://github.com/docker/compose/pull/5055.

diff -Nurp docker-compose-1.14.0.orig/compose/cli/colors.py docker-compose-1.14.0/compose/cli/colors.py
--- docker-compose-1.14.0.orig/compose/cli/colors.py	2017-03-18 18:47:14.000000000 -0500
+++ docker-compose-1.14.0/compose/cli/colors.py	2017-07-26 21:56:11.733902819 -0500
@@ -1,8 +1,6 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
-import colorama
-
 NAMES = [
     'grey',
     'red',
@@ -33,7 +31,6 @@ def make_color_fn(code):
     return lambda s: ansi_color(code, s)
 
 
-colorama.init(strip=False)
 for (name, code) in get_pairs():
     globals()[name] = make_color_fn(code)
 
diff -Nurp docker-compose-1.14.0.orig/setup.py docker-compose-1.14.0/setup.py
--- docker-compose-1.14.0.orig/setup.py	2017-06-06 14:33:59.000000000 -0500
+++ docker-compose-1.14.0/setup.py	2017-07-26 21:56:31.978904382 -0500
@@ -31,7 +31,6 @@ def find_version(*file_paths):
 
 install_requires = [
     'cached-property >= 1.2.0, < 2',
-    'colorama >= 0.3.7, < 0.4',
     'docopt >= 0.6.1, < 0.7',
     'PyYAML >= 3.10, < 4',
     'requests >= 2.6.1, != 2.11.0, < 2.12',
